embed/extmod/modtrezorcrypto/modtrezorcrypto-secp256k1_zkp.h

```diff
@@ -0,0 +1,247 @@
+/*
+ * This file is part of the TREZOR project, https://trezor.io/
+ *
+ * Copyright (c) SatoshiLabs
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "py/objstr.h"
+
+#include "ecdsa.h"
+#include "secp256k1.h"
+#include "secp256k1_ecdh.h"
+#include "secp256k1_preallocated.h"
+#include "secp256k1_recovery.h"
+
+STATIC uint8_t g_buffer[4500] = {0};
```

Comment:
We should decrease this to the right number. I think it's actually somewhere around 4200 bytes, so we can save a few hundred bytes.

Reply:
done